docs/tutorials/finetune.md

```diff
@@ -75,7 +75,7 @@ lr_config = dict(
     warmup_ratio=0.001,
     # [7] yields higher performance than [6]
     step=[7])
-total_epochs = 8  # actual epoch = 8 * 8 = 64
+runner = dict(max_epochs=8)
```

Comment:
Add comment here indicate that max_epochs and the step in lr_config need specifically tuned for the customized dataset.